mypy.ini

```diff
@@ -15,4 +15,5 @@ no_implicit_optional = True
 warn_return_any = True
 no_implicit_reexport = True
 strict_equality = True
-warn_redundant_casts = True
\ No newline at end of file
+warn_redundant_casts = True
+exclude = venv
```

Comment:
Given that `files =` is specified above, is this really needed?

Reply:
I keep running into this issue where if I just have `files = ` mypy still looks through the venv.

Reply:
no never. I just tried again and get the same problem: 
```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

venv/bin/jp.py:12: error: Function is missing a return type annotation  [no-untyped-def]
venv/bin/jp.py:54: error: Call to untyped function "main" in typed context  [no-untyped-call]
Found 2 errors in 1 file (checked 11 source files)
```

Reply:
Taking this to DM to debug.